### PR TITLE
getDefaultProvider implementation

### PR DESCRIPTION
**Pull request by joan41868**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/networks/src.ts/index.ts [hunk 1]
```diff
@@ -179,6 +205,23 @@ const networks: { [name: string]: Network } = {
 
     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+
+    // hedera networks
+    hederaMainnet: {
```
Comment (Daniel-K-Ivanov) [inline packages/networks/src.ts/index.ts:210]: lets remove the `testnet` one and change the behaviour of the default provider to:

- `hethers.providers.getDefaultProvider('mainnet' | 'testnet' | 'previewnet' | 290, 291, 292)`

File: packages/providers/src.ts/index.ts [hunk 2]
```diff
@@ -44,11 +44,11 @@ const logger = new Logger(version);
 // Helper Functions
 
 function getDefaultProvider(network?: Networkish, options?: any): BaseProvider {
-    if (network == null) { network = "homestead"; }
+    if (network == null) { network = "mainnet"; }
+    // @TODO: Add support for IpcProvider; maybe if it ends in ".ipc"?
```
Comment (Daniel-K-Ivanov) [inline packages/providers/src.ts/index.ts:48]: nit: remove TODO

File: packages/tests/src.ts/test-providers.ts [hunk 3]
```diff
@@ -1418,4 +1419,19 @@ describe("Test Hedera Provider", function () {
        // @ts-ignore
        assert.strictEqual(record.valid_duration_seconds, '120');
    });
+
+   it("Is able to get hedera provider as default", async() => {
```
Comment (Daniel-K-Ivanov) [inline packages/tests/src.ts/test-providers.ts:1423]: I think that we should add test for `ethers.providers.getDefaultProvider()` and verify that it connects to mainnet

File: packages/providers/src.ts/index.ts [hunk 4]
```diff
@@ -44,11 +44,11 @@ const logger = new Logger(version);
...
 
     // If passed a URL, figure out the right type of provider based on the scheme
     if (typeof(network) === "string") {
-        // @TODO: Add support for IpcProvider; maybe if it ends in ".ipc"?
 
         // Handle http and ws (and their secure variants)
         const match = network.match(/^(ws|http)s?:/i);
```
Comment (Daniel-K-Ivanov) [inline packages/providers/src.ts/index.ts:54]: question: I guess that we will handle this logic once we have the non-default provider?

**Review by Daniel-K-Ivanov — CHANGES_REQUESTED** [review]

File: packages/providers/src.ts/index.ts (hunk 4 above)
Comment (joan41868) [reply packages/providers/src.ts/index.ts:54]: Yes. I was thinking about implementing the `HederaProvider` which accepts 2 URLs right before doing this.

**Review by Daniel-K-Ivanov — APPROVED** [review]

**Pull request merged**